packages/terra-form/src/Control.jsx

```diff
@@ -82,16 +84,16 @@ const Control = ({
   value,
   ...customProps
 }) => {
-  const labelClassNames = classNames(
-    'terra-Form-control',
-    { 'terra-Form-control--inline': isInline },
+  const labelClassNames = cx([
+    'control',
+    { 'control-inline': isInline },
     customProps.className,
-  );
+  ]);
 
-  const labelTextClasses = classNames(
-    'terra-Form-control-label-text',
+  const labelTextClasses = cx([
+    'control-label-text',
     labelTextAttrs.className,
-  );
+  ]);
 
   const controlInputAttrs = Object.assign({}, inputAttrs);
```

Comment:
Good Eye! I completely missed that! It's fixed here. https://github.com/cerner/terra-core/pull/646/commits/8ab614c9491b5a2a96b418202f0aab54a2f5de3a